learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,94 @@
+
+![Meilisearch Cloud form to create a project complete, with "search-app" as the project's name and "Frankfurt" as the region](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/05.create-a-meilisearch-cloud-project-form.png)
+
+Once you click on **Create project**, you should see the following message: “Your Meilisearch + Vercel integration is one click away from being completed.” Click "Finish the Vercel integration setup". Meilisearch will then redirect you back to the Vercel integration page.
+
+![Meilisearch integration page in Vercel's dashboard](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/06.meilisearch-cloud-integration-page-in-vercel-dashboard.png)
+
+### Understand and use Meilisearch API keys
+
+Meilisearch creates two default API keys: [`Default Search API Key` and `Default Admin API Key`](/learn/security/master_api_keys#using-default-api-keys-for-authorization). 
+
+#### Admin API key
+
+Use the `Default Admin API Key`, to control who can access or create new documents, indexes, and change index settings. Be careful with the admin key and avoid exposing it in public environments.
+
+#### Search API key
+
+Use the `Default Search API Key` to access the [search route](/reference/api/search). This is the one you want to use in your front end.
+
+Both keys are automatically added to Vercel along with the Meilisearch URL.
+
+<Capsule intent="tip">
+The master key–which hasn’t been added to Vercel–grants users full control over an instance. You can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
+</Capsule>
+
+### Review your project settings
+
+Go back to your project settings and check the new Meilisearch environment variables:
+- `MEILISEARCH_ADMIN_KEY`
+- `MEILISEARCH_URL`
+- `MEILISEARCH_SEARCH_KEY`
+
+![Display the environment variables in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/07.project-settings-environment-variables-tab.png)
+
+## Take advantage of the Meilisearch Cloud dashboard
+
+![Meilisearch Cloud dashboard: overview of the "search-app" project](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/08.meilisearch-cloud-dashboard.png)
+
+Use the [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/), to index  documents and manage your project settings.
+
+## Resources and next steps
+
+Check out the [quick start guide](https://www.meilisearch.com/docs/learn/getting_started/quick_start#add-documents) for a short introduction on how to use Meilisearch. We also provide many [SDKs and tools](/learn/what_is_meilisearch/sdks), so you can use Meilisearch in your favorite language or framework. 
```

Comment:
why is this link to docs absolute instead of relative?

Reply:
Good catch, it must have slipped